databases/app-config-sync/fragment-feature-release-toggle-definitions.js

```diff
@@ -0,0 +1,37 @@
+{
+  channels: {
+    view: [ 'view-feature-release-toggle-definitions', 'view-config' ],
+    add: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    replace: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    remove: [ 'remove-feature-release-toggle-definitions', 'remove-config' ]
+  },
+  typeFilter: function(doc, oldDoc) {
+    // Example valid document IDs: "feature-release-toggle.foo-toggle", "feature-release-toggle.bar_1"
+    return /^feature-release-toggle\.[a-z0-9_-]+$/.test(doc._id);
```

Comment:
Keep in mind that the [featureReleaseToggles](https://github.com/Kashoo/kashoo-document-definitions/blob/master/databases/app-config-sync/fragment-feature-release-toggles.js) document type implies no restrictions on the characters that may be used in a feature toggle's name. This regular expression introduces an incompatibility. Either it should be relaxed to allow any character type or the `arrayElementsValidator` on the `enabledFeatures` property needs to be tightened to match this value.